propolis/src/hw/bhyve/atpic.rs

```diff
@@ -0,0 +1,35 @@
+use std::sync::Arc;
+
+use crate::dispatch::DispCtx;
+use crate::inventory::Entity;
+use crate::migrate::Migrate;
+
+use erased_serde::Serialize;
+
+pub struct BhyveAtPic {}
+impl BhyveAtPic {
+    pub fn create() -> Arc<Self> {
+        Arc::new(Self {})
+    }
+}
+
+impl Entity for BhyveAtPic {
+    fn type_name(&self) -> &'static str {
```

Comment:
Not specific to this PR, but we should just make this an associated const that's a `&'static str`.

Reply:
Is that possible with trait objects?  A `const fn` definitely wasn't.

Reply:
Ah right, that's true we do need it to be object safe.